src/psyclone/tests/psyir/nodes/omp_clause_test.py

```diff
@@ -151,6 +151,37 @@ def test_private_clause():
     assert private.clause_string == "private"
 
 
+def test_private_clause_create():
```

Comment:
I think you can parameterise this test so that it works for both `OMPPrivateClause` and `OMPFirstPrivateClause`. We can then do away with the almost identical test?

Reply:
Done